src/helpers/figures/charts/runtime/chartjs_show_values.ts

```diff
@@ -34,7 +34,9 @@ export function getPyramidChartShowValues(
     background: definition.background,
```

Comment:
nitpick: commit title is "[FIX] pyramid_chart: showValues plugin displays absolute values", but the thing you fix here is the display of the zeros.